README.md

```diff
@@ -494,17 +558,17 @@ Once is finished you can check the results on the Grafana Dashboard http://local
 
 Additionally, we have included a new Dashboard to show how to work with the Caching information of the task execution:
 
-![](resources/taskcache.png) 
+![](resources/taskcache.png)
 
 http://localhost:3003/d/WlpZEBRMz/task-cache-info?orgId=1
 
 
 
-## Other Plugins  <a name="plugins"></a>
+## Other Existing libraries  <a name="plugins"></a>
```

Comment:
Nit: for consistency in this title, we should either do:
```suggestion
## Other Existing Libraries  <a name="plugins"></a>
```
Or:
```suggestion
## Other existing libraries  <a name="plugins"></a>
```

Reply:
done